UI_DSM/UI_DSM.Client/ViewModels/Pages/Administration/ProjectPages/IProjectPageViewModel.cs

```diff
@@ -27,6 +32,21 @@ public interface IProjectPageViewModel
         /// </summary>
         IProjectDetailsViewModel ProjectDetailsViewModel { get; }
 
+        /// <summary>
+        ///     Value indicating if the <see cref="DxPopup" /> for the creation of <see cref="Participant" /> is visible
+        /// </summary>
+        bool CreationPopupVisible { get; set; }
```

Comment:
A little bit religious, but a ViewModel should not "know" anything about its View. A state that describes that a Creation Popup is visible should be implemented from the ViewModel's point of view. So maybe the name should be like "LockedForEdit", or something like that. I don't know what it exactly does.

Reply:
@lxatrhea Are the changes ok for you?